packages/tableofcontents/init.lua

```diff
@@ -221,16 +221,16 @@ end
 package.documentation = [[
 \begin{document}
 The \autodoc:package{tableofcontents} package provides tools for class authors to
-create tables of contents. When you are writing sectioning commands such
+create tables of contents (TOCs). When you are writing sectioning commands such
 as \code{\\chapter} or \code{\\section}, your classes should call the
 \autodoc:command{\tocentry[level=<number>, number=<strings>]{<title>}}
 command to register a table of contents entry.
-At the end of each page the class will call a hook function (\code{moveTocNodes}) that collates the table of contents entries from that pages and logs which page they’re on.
+At the end of each page the class will call a hook function (\code{moveTocNodes}) that collates the table of contents entries from that pages and records which page they’re on.
 At the end of the document another hook function (\code{writeToc}) will write this data to a file.
 The next time the document is built, any use of the \autodoc:command{\tableofcontents} (typically near the beginning of a document) will be able to read that index data and output the TOC.
 Because the toc entry and page data is not available until after rendering the document,
 the TOC will not render until at least the second pass.
-If by chance rendering the TOC itself changes the document pagination (e.g. the TOC spans more than one page) it might be necessary to run SILE 3 times to get accurate page numbers shown in the TOC.
+If by chance rendering the TOC itself changes the document pagination (e.g., the TOC spans more than one page) it might be necessary to run SILE three times to get accurate page numbers shown in the TOC.
```

Comment:
While we are at it: "three" times > "several" times? (as soon as there are other packages doing the same type of things, or more than one ToC, etc. this is going to be hard to predict!)

Reply:
Ah yes, in `c01-whatis.sil` I changed 'three' to 'several.' Missed this one.

Reply:
While theoretically possible, this was worded as 3 because it would be pretty tricky come up with an example that took 4. It would require the addition of a TOC to change the total page count so substantially that the raw size of the page numbers (e.g. number of digits) caused a third reflow of the document onto new pages even after the body of the ToC contents had been typeset with *close* page numbers once. If the number of pages taken up by your ToC content changes based on page numbers 999 overflowing to 1000 and now your TOC takes up 5 pages instead of 6, *AND* the overflow of 10000 to 1002 causes the ToC to take up 7 pages instead of 6 then you have bigger problems than number of SILE runs. This isn't "several", 3 is a magic number. 2 will probably work 80% of the time, but 3 will work cover the 50%-99.9999% of the time. If you have to go to four to get accurate results you should reconsider your life choices.

Reply:
The entry in c01 is unrelated to the ToC (and other packages that save references for future runs). That has to do with Knuth's line and breaking algorithm. We can gloss over that with "several" even though it is a fixed number too because the user doesn't need to do anything different based on that knowledge. For rerunning SILE to get accurate ToC page numbers they do need to know that 3 is the sweet spot.